src/main/java/Iris.java

```diff
@@ -0,0 +1,143 @@
+import java.util.Scanner;
+
+public class Iris {
+    public static Task[] tasks = new Task[100];
```

Comment:
For better scalability and flexibility, consider using ArrayList<Task> instead of a fixed-size array. This will eliminate the need to check if the array is full and allows dynamic resizing when adding tasks.